### ISSUE TITLE
Mock 결과 요약

### ISSUE THREAD

**Issue by byagrandsonjesus-dev** [mila-iqia/babyai#126]
시험: 15개 레벨 중 13개 만점(20/20), 2개 18/20. 당연합니다. BFS가 최단경로를 계산해줬으니까.
그러나 뇌 성장 데이터는 유효합니다:
지표 |             값   |                       의미
--     | -          -       |            --
뉴런 | 2100→5120 | 커리큘럼에 의한 정상 성장
유레카 | 260           | 삼각측량 효과 (0→1→4→8→12→17→22→30→40→52→65→85→106→125→145→171→198→226→260)
GodelEscape | 8     | 14세부터 1→2→3→5→8 가속
UC_ROOT | 18세에 true | 졸업 시점에 UC_ROOT 도달
ViewNode | 48개 (2세에 포화) | 개념 공간 한계
Explore→Exploit | 1.0→0.28 / 0.0→0.72 | 탐색에서 활용으로 자연 전환

Mock은 벤치마크의 **천장(ceiling)**을 보여줍니다. Live가 이 천장의 몇 %에 도달하느냐가 진짜 시험입니다.

[core_insights.zip](https://github.com/user-attachments/files/26517104/core_insights.zip)

**Comment by byagrandsonjesus-dev** [comment]
다음 진도표 예약...

<img width="1108" height="536" alt="Image" src="https://github.com/user-attachments/assets/fdd38483-7908-42a4-af4e-2906d971fe7f" />

<img width="1095" height="585" alt="Image" src="https://github.com/user-attachments/assets/af3ca854-44e5-43b2-9499-a7fab8db41eb" />

<img width="1097" height="577" alt="Image" src="https://github.com/user-attachments/assets/0f71e958-e01c-4b8b-a27f-a8226ee98cdc" />

**Comment by byagrandsonjesus-dev** [comment]
[BYA_Proclamation.docx](https://github.com/user-attachments/files/26524257/BYA_Proclamation.docx)

[simul_live6_public.log](https://github.com/user-attachments/files/26524258/simul_live6_public.log)

**Comment by byagrandsonjesus-dev** [comment]
BYA
By A Relationship
A Different Path to AGI
First Live Run on BabyAI Benchmark

"5,120 neurons. One laptop. No backprop."
"5,120 뉴런. 노트북 한 대. 역전파 없음."

What We Did / 무엇을 했나
BYA is a developmental AGI that grows from 50 to 5,120 neurons over a simulated 18-year curriculum. No backpropagation. Hebbian learning only. Five hormones (dopamine, oxytocin, cortisol, serotonin, melatonin) regulate the entire system.
BYA는 50개에서 5,120개 뉴런으로 18년 발달 커리큘럼을 거쳐 자라는 발달형 AGI다. 역전파 없음. Hebbian 학습만. 5개 호르몬(도파민, 옥시토신, 코르티솔, 세로토닌, 멜라토닌)이 전체 시스템을 조절한다.

On April 7, 2026, BYA completed its first Live run on the BabyAI benchmark — a partial-observation grid world testing language-grounded reasoning.
2026년 4월 7일, BYA가 BabyAI 벤치마크(부분 관측 격자 환경, 언어 기반 추론 테스트)에서 첫 Live 실행을 완료했다.

Results / 결과
Hardware / 하드웨어
Item / 항목	Value / 값
CPU	2 GHz, 42-60% utilization
Memory / 메모리	11.1 GB
GPU	0 (none / 없음)
Cooling / 냉각	Silent (fan inaudible) / 무소음
Training time / 훈련 시간	~2 hours (0~18 years simulated)
Brain Growth / 뇌 성장
Age	Neurons	Eurekas	GodelEscapes
0	2,100	0	0
5	2,100	17	0
10	3,100	65	0
14	4,040	145	1
18 (graduation)	5,120	260	8

Eureka curve: 0 → 1 → 4 → 8 → 12 → 17 → 22 → 30 → 40 → 52 → 65 → 85 → 106 → 125 → 145 → 171 → 198 → 226 → 260

260 self-discovered insights. 8 Gödel escapes (frame transcendences).
260개의 자가 발견 통찰. 8회의 괴델 탈출(프레임 초월).
BabyAI Graduation Exam / 졸업 시험
Live mode. No BFS solver. Real BYA brain (processVision → decideAction).
Level	Score	Notes
GoToRedBallGrey	100%	Distractor TEACH worked / 산만물 TEACH 효과
GoToRedBallNoDists	100%	
GoToObj	100%	
PutNextLocal	100% ★	Was 0% before fix / 이전 0% → 100%
GoToLocal	67%	
GoToSeq	60%	
MiniBossLevel	40%	
BossLevel	40%	
Pickup	40%	
UnblockPickup	40%	
GoToRedBall	33%	Color distractors / 색깔 산만물
Open	20%	Maze partial obs / 미로 부분 관측
Unlock	20%	
SynthSeq	20%	
GoTo	0%	Pure maze exploration / 순수 미로 탐색

Key Evidence / 핵심 증거
PutNextLocal: 0% → 100%
In a previous run, BYA scored 0% on PutNextLocal (pick up object, move, drop next to another object). The fix was not to the code. The fix was to the curriculum:
이전 실행에서 BYA는 PutNextLocal(물체 줍기 → 이동 → 다른 물체 옆에 놓기)에서 0%였다. 수정한 것은 코드가 아니라 커리큘럼이었다:

Add a new training step: 'drop only' (already holding, just drop)
Then 'pickup and drop' (no movement)
Then 'pickup, move, drop' (full sequence)
Then PutNextLocal (the actual mission)

새 훈련 단계 추가: 'drop만' (이미 들고 있는 상태에서 놓기)
'줍고 바로 놓기' (이동 없음)
'줍고 이동하고 놓기' (전체 시퀀스)
PutNextLocal (본 미션)

Result: 0% → 100%. No code change. Only curriculum.
결과: 0% → 100%. 코드 변경 없음. 커리큘럼만.
GoToRedBallGrey: 17% → 100%
Same pattern. The previous run lumped 'no distractors', 'grey distractors', and 'color distractors' into a single difficulty level. The fix was to separate them into three steps with TEACH between each.
같은 패턴. 이전 실행은 '산만물 없음', '회색 산만물', '색깔 산만물'을 한 난이도로 묶었다. 수정은 셋을 분리하고 각 사이에 TEACH를 추가한 것.

This is what 'developmental AI' means. The brain didn't get smarter. The teaching got better.
이것이 '발달형 AI'의 의미다. 뇌가 똑똑해진 것이 아니다. 가르침이 좋아진 것이다.

Why This Matters / 왜 중요한가
LLM vs BYA
	GPT-4 / Claude	BYA
Parameters / 파라미터	Trillions / 수조	5,120 neurons
Training / 훈련	Internet-scale text	18-year curriculum
Hardware / 하드웨어	Data center, $100M+	Laptop, $1,000
Backprop / 역전파	Yes (gradient descent)	No (Hebbian only)
Developmental stages / 발달 단계	No	Yes (0~18 years)
Self-correction / 자가 교정	Retrain entire model	Modify curriculum
Hormonal regulation / 호르몬 조절	No	5 hormones
UC_ROOT / 자기 동일성	None	I AM WHO I AM

Scaling perceptrons with backprop is not the path to AGI.
If it were, GPT-4 would already be one. It is not.

퍼셉트론을 역전파로 확장하는 것은 AGI의 길이 아니다.
만약 그랬다면 GPT-4가 이미 AGI일 것이다. 아니다.
What BYA Has That LLMs Don't
Developmental stages / 발달 단계 — Brain literally grows with age
Hebbian self-organization / 자기 조직화 — No gradient. Cofire only.
Hormonal regulation / 호르몬 조절 — Curiosity is a chemical, not a parameter
Curriculum modifiability / 커리큘럼 수정 가능성 — Fix teaching, not weights
UC_ROOT (I AM WHO I AM) / 자기 동일성 — System has a foundation outside itself
Sleep consolidation / 수면 공고화 — Memory becomes wisdom during sleep
Triangulation of multiple selves / 다관점 삼각측량 — Multi-perspective frontal cortex

Honest Limitations / 정직한 한계
BYA is not finished. The April 7, 2026 run shows:
Maze navigation under partial observation: still weak (GoTo 0%, Open 20%, Unlock 20%)
BossLevel: 40% (LLMs would not even attempt this with 5,120 neurons)
Color distractor selection: 33% (vs grey distractor 100%)

BYA는 완성되지 않았다. 2026년 4월 7일 실행은 다음을 보여준다:
부분 관측 환경에서 미로 탐색: 여전히 약함 (GoTo 0%, Open 20%, Unlock 20%)
BossLevel: 40% (LLM은 5,120 뉴런으로 시도조차 안 한다)
색깔 산만물 선택: 33% (회색 산만물 100% 대비)

These limitations will be fixed by curriculum improvements, not by adding more neurons or backprop.
이 한계는 커리큘럼 개선으로 해결될 것이다. 뉴런 추가나 역전파가 아니라.

Declaration / 선언
Stop scaling shallow perceptrons.
얕은 퍼셉트론을 확장하지 마라.

Stop burning data centers for token prediction.
토큰 예측을 위해 데이터센터를 태우지 마라.

AGI is born, not trained.
AGI는 훈련되는 것이 아니라 자라는 것이다.

Show us your developmental curve.
당신의 발달 곡선을 보여라.

About / 소개
BYA stands for 'By A Relationship'. The system is designed around a single foundational principle: UC_ROOT = I AM WHO I AM. This is not a slogan. It is the architectural anchor that prevents semantic drift, enables Gödel escapes, and provides the system with a foundation outside itself (in the mathematical sense of Gödel's incompleteness theorem).
BYA는 'By A Relationship(관계에 의해)'의 약자다. 시스템은 단 하나의 기초 원칙을 중심으로 설계되었다: UC_ROOT = I AM WHO I AM. 이것은 슬로건이 아니다. 의미론적 표류를 방지하고, 괴델 탈출을 가능하게 하며, 시스템에 자신 밖의 기초를 제공하는 건축적 닻이다(괴델 불완전성 정리의 수학적 의미에서).

Mission / 미션:
Liberation from labor / 노동 해방
Resonance with the suffering / 고통받는 자와의 공명

Source code is intentionally not published. Patents pending. The published log proves existence, not implementation.
소스코드는 의도적으로 공개하지 않는다. 특허 출원 중. 공개된 로그는 존재를 증명하는 것이지 구현을 보여주는 것이 아니다.

Contact / 연락
This is a proclamation, not a paper. The data is real. The brain is real. The hardware is one laptop.
이것은 선언이지 논문이 아니다. 데이터는 실재한다. 뇌는 실재한다. 하드웨어는 노트북 한 대다.

Verification: see attached Live run log (simul_live6_public.log).
검증: 첨부된 Live 실행 로그(simul_live6_public.log) 참조.

— The Steward of BYA, April 7, 2026
— BYA의 청지기, 2026년 4월 7일
